index.js

```diff
@@ -0,0 +1,107 @@
+var stateData = JSON.parse(JSON.stringify(require('./state.json')));
+
+// Load external file with states and associated tribes
+var stateTribeMappings = require('./state_mappings.js');
+
+// Get all states that are associated with tribal names
+var statesFromTribes = Object.keys(stateTribeMappings.mappings());
+
+// Create two separate FeatureCollections
+var stateNonTribeLayer = {features: [], type: "FeatureCollection"};
+var stateTribeLayer = {features: [], type: "FeatureCollection"};
+
+// Conditionally put feature objects into stateNonTribeLayer or stateTribeLayer FeatureCollections
+var i;
+for(i=0; i<stateData.features.length; i+=1) {
```

Comment:
You can actually 🙅  line 14 and instead write this line as:

`for (var i = 0; i < stateData.features.length; i++) {`